classes/class-paysoncheckout-for-woocommerce-confirmation.php

```diff
@@ -169,9 +169,12 @@ public function confirm_payson_order( $order_id ) {
 			$order->save();
 			$order->add_order_note( __( 'Payment via PaysonCheckout, order ID: ', 'payson-checkout-for-woocommerce' ) . $payment_id );
 			$order->payment_complete( $payson_order['purchaseId'] );
+
+			$order->add_order_note( __( 'Order confirmed on a callback from Payson.', 'woocommerce-gateway-payson' ) );
+			PaysonCheckout_For_WooCommerce_Logger::log( 'Order confirmed on a callback from Payson. Payment id: ' . $payment_id . ' Order id: ' . $order_id );
+
```

Comment:
This method isn't only called for callbacks, it is also called for regular payments that don't have to wait for a callback to happen. I believe this will always print this even if it was not caused by a callback right?